virttest/utils_test/qemu/__init__.py

```diff
@@ -725,6 +725,10 @@ def check_memory(self, vm=None):
         utils_misc.verify_dmesg()
         self.os_type = self.params.get("os_type")
         guest_mem_size = super(MemoryHotplugTest, self).get_guest_total_mem(vm)
+        # Notes:
+        #    for memory hotplug on windows guest, need to wait several seconds
```

Comment:
So you can update the note after your change? Not only windows.

Reply:
Agree.